docker/accumulo/docker-compose.yaml

```diff
@@ -62,19 +76,23 @@ services:
     - /data2
 
   accumulo-master:
-    depends_on:
-    - hdfs-namenode
-    - hdfs-datanode
-    - zookeeper
     image: gchq/accumulo:${ACCUMULO_VERSION}
+    depends_on:
+      hdfs-namenode:
```

Comment:
It waits for the HDFS namenode. This is because inside the Accumulo container entrypoint there's code which waits for the datanode to be ready and that the Hadoop FS is available. It just needs the namenode to be ready as it must be able to connect to this first.

It's not possible to have a healthcheck for the datanode itself because this would require using a Hadoop client which would require a lot of work. This isn't needed as the Accumulo container already checks this before starting up.